extensions/guacamole-auth-totp/src/main/resources/translations/fr.json

```diff
@@ -29,6 +29,24 @@
 
         "SECTION_HEADER_DETAILS" : "Détails:"
 
+    },
+
+    "USER_ATTRIBUTES" : {
+        
+        "FIELD_HEADER_GUAC_TOTP_DISABLED"      : "Désactiver TOTP :",
+        "FIELD_HEADER_GUAC_TOTP_KEY_GENERATED" : "Clé secrète générée :",
+        "FIELD_HEADER_GUAC_TOTP_KEY_CONFIRMED" : "Appareil d'authentification confirmé :",
```

Comment:
I think there's an extra space on each of these lines between the text and the ":".

Reply:
Extra spaces removed!